src/v/kafka/client/configuration.cc

```diff
@@ -33,7 +33,7 @@ configuration::configuration()
       "retries",
       "Number of times to retry a request to a broker",
       config::required::no,
-      5)
+      0)
```

Comment:
?

Reply:
I would prefer it if the defaults weren't changed. I think Noah is going to change how config is done, for now it might be less intrusive to do `kc::shard_local_cfg().retries.set_value(0);` at some point after the client is started.

Reply:
@senior7515 explanation is in the commit, i comment that this is temporary until we get a change in to manipulate config parameters on a more granular level, i.e. not project wide settings but maybe a constructor param or parameter to a method. @BenPope got it will fix

Reply:
Yup its in, resolving